api/controller.py

```diff
@@ -725,6 +727,7 @@ def search(self, lane_identifier):
         if isinstance(lane, ProblemDetail):
             return lane
         query = flask.request.args.get('q')
+        media = flask.request.args.get('media')
```

Comment:
This is where we need to convert between the schema.org URIs (e.g. "http://bib.schema.org/Audiobook") and the enumerated type values that `search` can handle (e.g. "audio"). The lookup table is `Edition.additional_type_to_medium`. If the incoming value for `media` isn't found in that lookup table, you should send an INVALID_INPUT problem detail document.

Sorry, I didn't remember this until I saw you pass in "audiobooks" in the test. That value should be rejected with a problem detail, and passing in "http://bib.schema.org/Audiobook" should convert to `Edition.AUDIO_MEDIUM` and pass that into your mock `search` method.